projects/harfbuzz/hb-subset-fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+timeout = 5
```

Comment:
Are you sure about that? i.e. any input taking more than 5 seconds to process would be reported as a timeout. By default we use 25 seconds.